.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  Mambaforge-pytest:
+    name: Unit-Test (${{ matrix.os }}, Mambaforge)
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8"]
+        os: ["ubuntu", "macos"]
+        include:
+          - os: ubuntu
+            environment-file: environment.yml
+            miniforge-variant: Mambaforge
+            miniforge-version: 4.9.2-4
+          - os: macos
+            environment-file: environment.yml
+            miniforge-variant: Mambaforge-pypy3
```

Comment:
this is cool 🤩 I haven't updated any of my CI since I've been using mamba so I'm excited to see this in action.

Reply:
Big fan of this! If we need to add in other files as time goes on, we can